internal/oauth2/oauth2.go

```diff
@@ -84,6 +84,8 @@ type ClientConfig struct {
 	TLSCert                string
 	TLSKey                 string
 	TLSRootCA              string
+	DPoP                   bool
+	DPoPSigningKey         string
```

Comment:
can we use the existing signing key instead?

Reply:
done